tracks/go/exercises/two-fer/README.md

```diff
@@ -0,0 +1,31 @@
+# Two Fer
+
+_Two Fer_ is the first core exercise after _Hello, World_. _Hello, World_ is auto-approved, so _Two Fer_ is the first exercise where someone will be blocked until a mentor approves their exercise, and also likely the first feedback they've ever gotten on their Go code.
+
+This is also going to be the most common exercise that we are asked to mentor, since some people will drop off after submitting just the one.
+
+We don't have any clear guidelines around when to approve. Some people have been adding comments and approving straight away, others have been approving when the solution has a nice, Go-ish shape (a small conditional that isolates the smallest difference, and a single line of string formatting).
+
+One mentor recommended to focus on a single thing, and only add other improvements as "food for thought" that the author can take with them as they move on to the next exercise.
+
+**The Conditional**
+* Is there duplicated content? Sometimes this will be two branches, sometimes this will be one branch with the whole phrase in it, and then the phrase repeated after the conditional. Here you can suggest getting the smallest possible difference isolated into the conditional.
+* Are they re-assigning the param to a local variable? You can point out that they have two variables doing the same thing, e.g. `input` and `value` are both doing the same thing. Can you get away with just having one of them?
+
+**String Concatenation**
+* are they using a slice and strings.Join()? Point them towards `fmt` package as the more conventional choice here. Maybe not point straight to which function, it's often useful to let people explore a bit and find it on their own.
+* are they using `+` to concatenate? Same thing, but a more gentle pointer since it's perfectly legit to see `+`, just more common to see `fmt`, and worth getting used to.
```

Comment:
Depending on how it is written, using `+` is also far, far faster than `fmt`. I tend to reserve `fmt` for times I need it, as it is comparatively expensive.